docs/development.md

```diff
@@ -0,0 +1,119 @@
+Development
+
+This document explains how to get started to develop the apisix-java-plugin-runner.
+
+Prerequisites
+-------------
+
+* JDK 8
+* APISIX 2.6.0
+* Clone the [apisix-java-plugin-runner](https://github.com/apache/apisix-java-plugin-runner) project.
+* Refer to [Debug](how-it-works.md#debug)  to build the debug environment.
+
+Install
+-------
+
+```shell
+cd /path/to/apisix-java-plugin-runner
+./mvnw install
+```
+
+Write Filter
+------------
+
+Refer to the code in the [sample](https://github.com/apache/apisix-java-plugin-runner/tree/main/sample)
```

Comment:
The sample changes data with hardcoded values. Better to show how can a plugin access the configuration.

Reply:
update, here https://github.com/apache/apisix-java-plugin-runner/pull/19/files#diff-ed24963c66d69025a9cef9bd7407fc07a29e397a71be4602c96d6a03f8d42274R63-R78